### ISSUE TITLE
AUC in the Classification on imbalanced data tutorial 

### ISSUE THREAD

**Issue by yoheimiyamoto** [tensorflow/tensorflow#51437]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:
https://www.tensorflow.org/tutorials/structured_data/imbalanced_data#check_training_history_2

## Description of issue (what needs changing):
In the documentation, the resampled model has the highest AUC, as shown below, but [the chart](https://www.tensorflow.org/tutorials/structured_data/imbalanced_data#plot_the_roc_3) shows that the baseline AUC is the highest. I think the AUC calculated by model.evaluate method may be incorrect.

>Here you can see that with class weights the accuracy and precision are lower because there are more false positives, but conversely the recall and AUC are higher because the model also found more true positives.

```
# AUC calculated by model.evaluate method

# baseline
0.9296237826347351

# weighted
0.9428448677062988

# resampled
0.9575912952423096
```

I have tried other methods to check the correct value. The result of calculating the AUC using the sklearn.metrics.roc_auc_score method is as follows. As expected, it can be confirmed that the AUC of the baseline is the highest.

```
# AUC calculated by sklearn.metrics.roc_auc_score method

baseline_auc = sklearn.metrics.roc_auc_score(test_labels, test_predictions_baseline) 
weighted_auc = sklearn.metrics.roc_auc_score(test_labels, test_predictions_weighted)  
resampled_auc = sklearn.metrics.roc_auc_score(test_labels, test_predictions_resampled) 

# baseline
0.9685415795364084

# weighted
0.9387766618590307

# resampled
0.9665411665226982
```

### Clear description

For example, why should someone use this method? How is it useful?

### Correct links

Is the link to the source code correct?

### Parameters defined

Are all parameters defined and formatted correctly?

### Returns defined

Are return values defined?

### Raises listed and defined

Are the errors defined? For example,
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/feature_column/categorical_column_with_vocabulary_file#raises

### Usage example

Is there a usage example?

See the API guide: https://www.tensorflow.org/community/contribute/docs_ref
on how to write testable usage examples.

### Request visuals, if applicable

Are there currently visuals? If not, will it clarify the content?

### Submit a pull request?

After confirming this issue with other people, I will make a pull request.

**Comment by sushreebarsa** [comment]
@yoheimiyamoto In order to reproduce the issue reported here,Could you please fill the issue [template](https://github.com/tensorflow/tensorflow/issues/new/choose),
Thanks!

**Comment by yoheimiyamoto** [comment]
@sushreebarsa 
>In order to expedite the trouble-shooting process here,Could you please fill the issue template

Done. Thanks.
